dependencies.yaml

```diff
@@ -62,6 +62,11 @@ runtime-deps:
     - python-m2crypto
     - gettext
     - python3-openssl
+    - libldap2-dev
+    - libsasl2-dev
+    - libssl-dev
+    - swig
+    - gcc
```

Comment:
Why do we need those new dependencies?

Reply:
As we are moving to pip, when you install `pam` or `m2crytpo`, the packages has dependencies outside of python, usually c libraries